### PR TITLE
Fixes oc cluster fails to abort when it first fails to pull an image

### LINKED ISSUE #16735: oc cluster fails to abort when it first fails to pull the ose image
https://gist.github.com/thoraxe/cc212c2aed681641bd3658dd1454f4c7

oc version = oc v3.7.0-0.127.0
kubernetes v1.7.0+80709908fd
features: Basic-Auth GSSAPI Kerberos SPNEGO

### PR DESCRIPTION
**Pull request by coreydaley**
Fixes #16735

### CODE REVIEW

**Comment by coreydaley** [comment]
@openshift/sig-developer-experience ptal

Please see comments in #16735 for details

File: pkg/oc/bootstrap/docker/dockerhelper/helper.go [hunk 1]
```diff
@@ -157,6 +157,13 @@ func (h *Helper) CheckAndPull(image string, out io.Writer) error {
 	if err != nil {
 		return starterrors.NewError("error pulling Docker image %s", image).WithCause(err)
 	}
+
+	_, _, err = h.client.ImageInspectWithRaw(image, false)
```
Comment (bparees) [inline pkg/oc/bootstrap/docker/dockerhelper/helper.go]: put a comment in about why we're doing this.

**Comment by bparees** [comment]
one nit and lgtm

File: pkg/oc/bootstrap/docker/dockerhelper/helper.go (hunk 1 above)
Comment (coreydaley) [reply pkg/oc/bootstrap/docker/dockerhelper/helper.go]: Updated

**Comment by coreydaley** [comment]
@bparees ptal, comment added

**Pull request merged**